storage/migrations/32_block_meta.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE chain.blocks
+    ADD COLUMN proposer_entity_id base64_ed25519_pubkey,
+    ADD COLUMN signer_entity_ids base64_ed25519_pubkey[];
```

Comment:
What was the reasoning for storing Entity IDs instead of Node IDs here? 

A validator entity can have multiple validator nodes registered. In that case, we lose information about which exact node signed the block. 

In practice, only a single node will be active at any given epoch (since the network limits to one at this point), so this shouldn't be that problematic. However, I feel we could easily have used node IDs here to avoid this.

Reply:
Actually, it probably makes sense/is fine, since we mostly don't care about individual validator nodes in other places/in the API.

Reply:
yeah, it's for reasons of API simplicity like that. the general style of nexus is to have analysis precompute stuff that the API side will need, even sometimes at the cost of lossiness

Reply:
> we mostly don't care about individual validator nodes in other places/in the API.

Yeah it was a combination of this, also because we don't track historical validator entity<>node mappings (though it is possible to add now in the `history.validators` table).